internal/cli/atlas/livemigrations/validation/create.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validation
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/openlyinc/pointy"
+	"github.com/spf13/cobra"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+type CreateOpts struct {
```

Comment:
just a heads up that validate and create use the same properties, when create comes into play it would be a good idea to have a helper struct you can embed on both

Reply:
I'll split when making the validate method (rather than early optimisation)

Reply:
yep, just wanted to give the heads up is something when we have two "identical" commands no need for it now